src/cli_plugin/install/index.js

```diff
@@ -18,6 +19,25 @@ function processCommand(command, options) {
   install(settings, logger);
 }
 
+function getDefaultConfigPath() {
+  const paths = [
+    fromRoot('config/kibana.yml'),
+    '/etc/kibana/kibana.yml'
+  ];
+
+  let defaultPath = paths[0];
```

Comment:
this also seems a little weird. Why are you initializing defaultPath to paths[0] if it is one of the paths that you're going to be examining in the following loop?